tests/test_mssqlcliclient.py

```diff
@@ -202,6 +203,37 @@ def test_stored_proc_multiple_result_sets(self):
         finally:
             shutdown(client)
 
+    
+    def test_select_result_unicode_encoded(self):
+        """
+            Verify the column names and string values in rows returned by select statement are properly encoded as unicode.
+        """
+        db_name = 'db_' + random_str()
+        table_name = 'table_' + random_str()
+        try:
+            client = create_mssql_cli_client()
+            setup_query = u"CREATE DATABASE {0};"\
+                u"USE {0};"\
```

Comment:
This test class assumes an AdventureWorks connection. Can we reuse the existing database?

Reply:
Creating a test database is removed from the unit test code.